package_manifest.yaml

```diff
@@ -113,6 +113,7 @@ foreman_scl_packages:
       releasers:
         - koji-foreman
         - koji-foreman-plugins
+    rubygem-ffi: {}
```

Comment:
We already have FFI, this shouldn't be needed.

Reply:
I've checked and it's not in manifest. So it doesn't need to be in `package_manifest.yaml`? or will it get refreshed automatically at some point?

Reply:
It's under the foreman proxy packages, but I think that's wrong:
https://github.com/theforeman/foreman-packaging/blob/d6c2dcb38d6ae0aad3ecbe3b1c8261eaed58b30a/package_manifest.yaml#L599

@ehelms you changed this in 3c6439a288d21e93f02efb7481048a4d19fe52fd but `rubygem-sassc` also depends on it for example.

Reply:
right, I've removed it from there:
https://github.com/theforeman/foreman-packaging/pull/4379/files#diff-9297bc45c5e9bdf810aa55f6c91853dcL588
looks better now?

Reply:
I think so. @ehelms?

Reply:
Yep, looks good.